src/config.rs

```diff
@@ -191,6 +221,7 @@ impl ConfigBuilder {
     ///     .set_time_format_custom(format_description!("[hour]:[minute]:[second].[subsecond]"))
     ///     .build();
     /// ```
+    #[cfg(not(feature = "chrono"))]
     pub fn set_time_format_custom(
         &mut self,
         time_format: &'static [FormatItem<'static>],
```

Comment:
So instead of adding `set_time_format` and `set_time_format_str`, only this parameter could change to something like `time_format: impl Into<Cow<'static, str>>`.